neqo-transport/src/recovery.rs

```diff
@@ -357,6 +356,14 @@ impl CongestionControl {
         }
     }
 
+    fn remove_ignore(&mut self, ignore_packets: &[SentPacket]) {
```

Comment:
```suggestion
    fn remove_ignored(&mut self, ignore_packets: &[SentPacket]) {
```

But I think that we should name this "discard_all()" instead, consistent with your other changes.

```suggestion
    fn discard_all(&mut self, ignore_packets: &[SentPacket]) {
```